spec/rails/loaders/yaml_spec.rb

```diff
@@ -21,6 +21,22 @@
     )
   end
 
+  context "when no top environmental keys present" do
```

Comment:
Let's also add an example when we have a single known environment present but not matching the current env (say, `production`).